Tests/iOS/Specs/DataStructures/ExpirySpec.swift

```diff
@@ -16,12 +16,12 @@ class ExpirySpec: QuickSpec {
           expect(expiry.date).to(equal(date))
         }
 
-        it("returns date by adding time interval") {
-          let date = NSDate().dateByAddingTimeInterval(1000)
-          expiry = .Seconds(1000)
-
-          expect(expiry.date.timeIntervalSince1970) ≈ (date.timeIntervalSince1970, 0.01)
-        }
+//        it("returns date by adding time interval") {
+//          let date = NSDate().dateByAddingTimeInterval(1000)
+//          expiry = .Seconds(1000)
+//
+//          expect(expiry.date.timeIntervalSince1970) ≈ (date.timeIntervalSince1970, 0.01)
+//        }
```

Comment:
It doesn't compile using latest Nimble version available, but this test could be rewritten using:

```swift
expect(expiry.date.timeIntervalSince1970).to(beCloseTo(date.timeIntervalSince1970, within: 0.01))
```

Nevertheless I would close this pull request as I'm not the author of the code to be merged and I haven't tested it properly.

I've been working on my own proposal to add Swift 2.3 support and I'll open a new pull request with it as soon as it passes tests properly.